articles/Getting-started-with-windows-forms-using-c-#/index.md

```diff
@@ -0,0 +1,176 @@
+
+With *windows forms*, you can make basic applications like a calculator to high-level applications such as a library database management application that requires a connection to the database from the application.
+### Prerequisites
+1. A basic understanding of c# programming language.
+2. Visual studio installed on your system.
+
+If you don't have Visual Studio installed on your computer. you can download it from [Here](https://visualstudio.microsoft.com/downloads/).
+
+### Make a new project
+In this tutorial, we will make a simple calculator! Make a new project by opening the visual studio, then click on File -> New -> Project -> Visual c# -> Windows desktop and choose Windows Forms App (.NET Framework).
+
+### ToolBox
+The *Toolbox* window provides you with all the controls that help you build your project inside the form. to display the Toolbox click on View -> Toolbox, or you can press Ctrl+Alt+X. To use one of the controls double click on one of them or simply drag and drop it.
+
+### Properties window
+The *Properties window* is used to show the properties of the form or the control that you click on, you can manipulate the properties of any control used in your form. To display the window you can click on View -> properties window or you can simply click F4.   
+
+![properties window and toolbox](properties window and toolbox.png)
+
+### Building the form with the controls
+First of all, let's start with the form and change the text to My calculator and the name to Cal_Form from the properties window.
+
+![Form_name_text](Form_name_text.png).
+
+
+**NOTE** that changing the name of a Control or the form will only change the name of the object inside the code, and that makes it easier to figure out which button or witch textbox you are using or editing right now, on the other hand changing the text of a Control will only change what appears to the user.
+
+Now let's begin by adding the buttons and the textbox. First, we will add the numbers as buttons then we will add the C button the Calculating button the Operation buttons, and finally, the Textbox that will show the answer. To add a Control, you can drag & drop it from the toolbox to the form or simply double click on it. Repeat this with remaining controls.
+
+![Controls_display](Controls_display.png).
+
+Don't forget to change both the controls *Texts* to numbers & operations and the *Name* of buttons & textbox that will change the object name inside the code like we previously did with the form.
+
+![First_look1](First_look1.png).
+
+### Let's start coding
+You can double click on any of the buttons and it will take you to the method where you can write your code inside of it, let's start with the numbers.
+Each one of these buttons will display a number on the textbox that we previously added.
```

Comment:
You can explain it better. Like,

When you click a button, the value of the button must get appended to the text box's value, which we will use later for calculating. 

Once we update the value of the textbox, it will be visually updated in the textbox.

Reply:
Something like this.